test/testKillImage.sh

```diff
@@ -5,7 +5,7 @@
 
 launchCommand="./pharo-launcher.sh image launch ./../shared/PharoLauncher.image"
 killCommand="./pharo-launcher.sh image kill --all"
-processListCommand="./pharo-launcher.sh image ps"
+processListCommand="./pharo-launcher.sh image processList"
```

Comment:
I didn't know that (not familiar with Docker), for me it looked like bash specific command :) Is our assumption that users of this API would be similar user base as Docker users? I changed that because API uses English words also for other commends (kind of consistency). Let me know, I can revert this back, if you'd rather prefer `ps`.

Reply:
We can keep it. Users of this command will tell us if processList takes too much time to type or if they prefer a shorcut